proxy/proxy.go

```diff
@@ -357,6 +372,11 @@ func proxyMain() {
 		os.Exit(1)
 	}
 	proxy.serve()
+
+	// Wait for all the goroutines started by helloHandler to finish. Not
+	// stricly necessary, as the process is about to exit, but it's eye
+	// please to match wg.Add() by the corresponding wg.Wait()
```

Comment:
Ok. My comment was simply to suggest a rewording as it currently doesn't parse fully in my mind :)